ee/localserver/krypto-ec-middleware.go

```diff
@@ -169,3 +169,25 @@ func extractChallenge(r *http.Request) (*challenge.OuterChallenge, error) {
 
 	return challenge.UnmarshalChallenge(decoded)
 }
+
+type bufferedHttpResponse struct {
+	header http.Header
```

Comment:
could probably simplify by embedding http.ResponseWriter and override Write() to write to buf